docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -60,14 +60,15 @@ Use the get snapshot API to return information about one or more snapshots, incl
 `<repository>`::
 (Required, string)
 Comma-separated list of snapshot repository names used to limit the request.
-Wildcard (`*`) expressions are supported.
+Wildcard (`*`) expressions are supported including combining wildcards with exclude patterns starting in `-`.
 +
 To get information about all snapshot repositories registered in the
 cluster, omit this parameter or use `*` or `_all`.
 
 `<snapshot>`::
 (Required, string)
-Comma-separated list of snapshot names to retrieve. Also accepts wildcards (`*`).
+Comma-separated list of snapshot names to retrieve. Also accepts wildcards (`*`) as well as combining wildcards and exclude patterns
+starting in `-`.
```

Comment:
```suggestion
starting with `-`.
```

Hopefully this is correct in English :)

Reply:
I think both work, but I adjusted to `with` now, seems more standard :)